Lib/asyncio/base_events.py

```diff
@@ -1306,8 +1306,12 @@ def _check_sendfile_params(self, sock, file, offset, count):
 
             exceptions = []
 
-            if reuse_address is None:
-                reuse_address = os.name == 'posix' and sys.platform != 'cygwin'
```

Comment:
Should we have a similar default for `reuse_port` or should users have to explicitly specify it?

```python
if reuse_port is None:
    reuse_port = hasattr(sockets, 'SO_REUSEPORT')
...
```

Personally, I'm in favor of having `reuse_port` as the default on support platforms, but either would be fine.